docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -69,17 +69,26 @@ test: manifests generate fmt vet setup-envtest ## Run tests.
 # The default setup assumes Kind is pre-installed and builds/loads the Manager Docker image locally.
 # CertManager is installed by default; skip with:
 # - CERT_MANAGER_INSTALL_SKIP=true
-.PHONY: test-e2e
-test-e2e: manifests generate fmt vet ## Run the e2e tests. Expected an isolated environment using Kind.
+.PHONY: setup-test-e2e
+setup-test-e2e: ## Set up a Kind cluster for e2e tests if it does not exist
 	@command -v $(KIND) >/dev/null 2>&1 || { \
 		echo "Kind is not installed. Please install Kind manually."; \
 		exit 1; \
 	}
-	@$(KIND) get clusters | grep -q 'kind' || { \
-		echo "No Kind cluster is running. Please start a Kind cluster before running the e2e tests."; \
+	$(KIND) create cluster --name project-test-e2e --wait 5m
+
+.PHONY: test-e2e
+test-e2e: setup-test-e2e manifests generate fmt vet ## Run the e2e tests. Expected an isolated environment using Kind.
+	go test ./test/e2e/ -v -ginkgo.v
+	$(MAKE) teardown-test-e2e
+
+.PHONY: teardown-test-e2e
+teardown-test-e2e: ## Tear down the Kind cluster used for e2e tests
+	@command -v $(KIND) >/dev/null 2>&1 || { \
+		echo "Kind is not installed. Please install Kind manually."; \
 		exit 1; \
 	}
```

Comment:
```suggestion
```
We already checked it to create the cluster, do we need to check if the kind exists to delete the cluster?
Could we not simplify the code by making it less verbose?